src/libraries/TickBitmap.sol

```diff
@@ -76,26 +76,25 @@ library TickBitmap {
 
     /// @notice Returns the next initialized tick contained in the same word (or adjacent word) as the tick that is either
     /// to the left (less than or equal to) or right (greater than) of the given tick
-    /// @param self The mapping in which to compute the next initialized tick
-    /// @param tick The starting tick
+    /// @param bitmap the tick bitmap compute the next initialized tick
+    /// @param bitPos The bit position of `bitmap` for the tick in question
+    /// @param compressed The compressed tick value, tick / tickSpacing rounded towards negative infinity
     /// @param tickSpacing The spacing between usable ticks
     /// @param lte Whether to search for the next initialized tick to the left (less than or equal to the starting tick)
     /// @return next The next initialized or uninitialized tick up to 256 ticks away from the current tick
     /// @return initialized Whether the next tick is initialized, as the function only searches within up to 256 ticks
     function nextInitializedTickWithinOneWord(
-        mapping(int16 => uint256) storage self,
-        int24 tick,
+        uint256 bitmap,
+        uint8 bitPos,
+        int24 compressed,
```

Comment:
why do we want to expose `compressed` at all? seems like we can just always compute compressed inside this helper? and potentially makes this function more prone to integration issues if they dont know what the correct value is for `compressed`